### PR TITLE
Fix exploding barrels not placing decals correctly

### LINKED ISSUE #361: Exploding barrels don't leave a mark on the ground anymore

### PR DESCRIPTION
**Pull request by toberge**
Closes #361

### CODE REVIEW

File: Assets/Scripts/Interactables/ExplodingBarrel.cs [hunk 1]
```diff
@@ -11,33 +11,40 @@ public class ExplodingBarrel : MonoBehaviour
...
 
     private void LeaveMark()
     {
         // TODO This assumes that down is down on all maps. Refactor if gravity is made to work differently sometime.
-        if (!Physics.Raycast(transform.position, Vector3.down, out RaycastHit hit, 3, explosionMarkMask))
+        // The raycast should start approximately in the center of the barrel, thus the + transform.up
+        if (!Physics.Raycast(transform.position + transform.up, Vector3.down, out var hit, explosionController.Radius, explosionMarkMask))
             return;
+
+        Debug.Log($"Hit at {hit.point}");
```
Comment (Fueredoriku) [inline Assets/Scripts/Interactables/ExplodingBarrel.cs:45]: Maybe remove debug?

**Review by Fueredoriku — CHANGES_REQUESTED** [review]
It seems there is a problem with the updated explodingBarrel code:
image

**Comment by toberge** [comment]
The issue should be fixed now 😅

**Review by Fueredoriku — APPROVED** [review]
Great work!

**Pull request merged**